tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -1736,6 +1736,47 @@ static struct device_info boards[] = {
 		.last_sysupgrade_partition = "file-system"
 	},
 
+	/** Firmware layout for the RE450 v3 */
+	{
+		.id     = "RE450-V3",
+		.vendor = "",
+		.support_list =
+			"SupportList:\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:00000000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:55530000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:45550000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:4A500000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:43410000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:41550000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:41530000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:4B520000}\r\n"
+			"{product_name:RE450,product_ver:3.0.0,special_id:42520000}\r\n",
+		.support_trail = '\x00',
+		.soft_ver = NULL,
+
+		/* We're using a dynamic kernel/rootfs split here */
+		.partitions = {
+			{"fs-uboot", 0x00000, 0x20000},
+			{"default-mac", 0x20000, 0x00020},
+			{"pin", 0x20020, 0x00020},
+			{"product-info", 0x21000, 0x01000},
+			{"partition-table", 0x22000, 0x02000},
+			{"soft-version", 0x24000, 0x01000},
+			{"support-list", 0x25000, 0x01000},
+			{"profile", 0x26000, 0x08000},
+			{"user-config", 0x2e000, 0x10000},
+			{"default-config", 0x3e000, 0x10000},
+			{"config-info", 0x4e000, 0x00400},
+			{"firmware", 0x50000, 0x7a0000},
+			{"radio", 0x7f0000, 0x10000},
+
```

Comment:
Please remove this empty line (and remove it for v2 as well while you are at it ...)

Reply:
Done.